tools/dedupe-types.js

```diff
@@ -20,7 +20,7 @@ import fs from "node:fs";
 //-----------------------------------------------------------------------------
 
 const importRegExp =
-	/^\s*\*\s*@import\s*\{\s*(?<ids>[^,}]+(?:\s*,\s*[^,}]+)*)\s*\}\s*from\s*"(?<source>[^"]+)"/u;
+	/^\s*\*\s*@import\s*\{\s*(?<ids>[^,}\s]+(?:\s*,\s*[^,}\s]+)*)\s*\}\s*from\s*"(?<source>[^"]+)"/u;
```

Comment:
```suggestion
	/^\s*\*\s*@import\s*\{(?<ids>[^,}]+(?:,[^,}]+)*)\}\s*from\s*"(?<source>[^"]+)"/u;
```

Same as https://github.com/eslint/json/pull/128#discussion_r2296760415